include/cqerl_protocol.hrl

```diff
@@ -36,11 +36,11 @@
 }).
 
 -record(cqerl_query_parameters, {
-    consistency         = ?CQERL_CONSISTENCY_ANY :: ?CQERL_CONSISTENCY_ANY .. ?CQERL_CONSISTENCY_LOCAL_SERIAL,
+    consistency         = any :: any | one | two | three | quorum | all | local_quorum | each_quorum | serial | local_serial | integer(),
```

Comment:
Do you think we could factor out the `?CQERL_CONSISTENCY_ANY .. ?CQERL_CONSISTENCY_LOCAL_SERIAL` typespec into a type and use that as an alternative to the atoms?